db/migrate/20240208124208_add_payment_url_to_forms.rb

```diff
@@ -0,0 +1,5 @@
+class AddPaymentUrlToForms < ActiveRecord::Migration[7.1]
+  def change
+    add_column :forms, :payment_url, :string
```

Comment:
Should add a validation that it's a URL?

Reply:
I've added this in the model, brought in the `validate_url` gem which we're also using in admin